### PR TITLE
Add Persian scale

### PR DESCRIPTION
**Pull request by mamins1376**
one of the widely used scales in Iranian music.

### CODE REVIEW

**Comment by mamins1376** [comment]
sorry, fixed.

File: src/core/InstrumentFunctions.cpp [hunk 1]
```diff
@@ -136,7 +136,7 @@ InstrumentFunctionNoteStacking::ChordTable::Init InstrumentFunctionNoteStacking:
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Half-Whole Diminished" ), { 0, 1, 3, 4, 6, 7, 9, 10, -1 } },
 	
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "5" ), { 0, 7, -1 } },
-	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Persian" ), { 0, 1, 4, 5, 7, 8, 10, -1 } }
+	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Persian" ), { 0, 1, 4, 5, 6, 8, 11, -1 } }
 } ;
```
Comment (zonkmachine) [inline src/core/InstrumentFunctions.cpp]: I liked this one too: { 0, 1, 4, 5, 7, 8, 10, -1 } and tracked it down.
https://en.wikipedia.org/wiki/Phrygian_dominant_scale
I suggest adding it too as **Phrygian dominant** or **Phrygian major** .

**Comment by mamins1376** [comment]
added as Phrygian dominant. any other suggestions are welcome!

**Comment by zonkmachine** [comment]
Cool! Squash it down to one commit?

**Comment by grejppi** [comment]
> Cool! Squash it down to one commit?

Why?

kuvakaappaus 2016-09-23 15-34-03

**Comment by mamins1376** [comment]
I manually squashed them, cause some of commits were too small to be a useful commit.

**Comment by Umcaruje** [comment]
> I manually squashed them, cause some of commits were too small to be a useful commit.

Yeah there is really no need to do that now anymore, and it can only add confusion. We can squash on merge now, so no matter how many commits you had, it all gets squashed down to one automatically.

Merging.

**Pull request merged**